### DISCUSSION TITLE
[Taskbar Thumbnail Reorder] requires disable/enable to work

### DISCUSSION THREAD

**Discussion by jibbers42** [ramensoftware/windhawk-mods#63] [Q&A]
After rebooting, I can't reorder taskbar thumbnail unless I open Windhawk and disable then enable Taskbar Thumbnail Reorder. Has anyone seen this? Is there a known fix? Should I create an issue?
Thanks

**Comment by sluggogle** [comment]
I love this feature and it works as expected on my side (Windows 11).
I do have to open Windhawk of course.

Open an issue or try to reach someone from the project on discord.

**Reply by jibbers42** [reply]
Yeah, the feature is great! I can't believe this isn't something MS has ever done built-in. To clarify, Windhawk is set to run on Windows startup - so it is running. The module needs to be disabled and then enabled to work.

I opened an issue a few days after creating this discussion at https://github.com/ramensoftware/windhawk-mods/issues/64.

**Comment by m417z** [comment]
Fixed in version 1.0.3 of the mod.
